### PR TITLE
Add Alphaday to Showcase

### PR DESCRIPTION
**Pull request by Mikael337**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by Mikael337** [comment]
Hello,

Can I get some help to resolve issues regarding checks? I did sign the CLA here: https://github.com/dfinity/cla/issues/429
I also did link our website in the submission but the website check gives an error.
Note: We are already grantees of ICP and have built the free-to-use dashboard for your community. We are just seeking a bit of help spreading awareness.
Thank you,
Mike

File: showcase.json [hunk 1]
```diff
@@ -3296,5 +3296,29 @@
...
     "twitter": "https://twitter.com/quezarHQ",
     "submittableId": "41431540"
+  },
+  {
+  "name": "Alphaday",
+  "description": "Alphaday is a powerful crypto workflow aggregator that pulls in news, on-chain data, social media, web3 services and much more into one customizable UI.",
+  "website": "https://app.alphaday.com/b/dfinity/",
+  "logo": "/img/showcase/alphaday_logo.webp",
+  "screenshots": [
+    "/img/showcase/alphaday_screenshot.webp"
```
Comment (jennifertrin) [inline showcase.json:3306]: Can you please add a screenshot file to this path?

File: showcase.json [hunk 2]
```diff
@@ -3296,5 +3296,29 @@
     "github": "https://github.com/QuezarHQ/pingICP",
     "twitter": "https://twitter.com/quezarHQ",
     "submittableId": "41431540"
+  },
+  {
+  "name": "Alphaday",
+  "description": "Alphaday is a powerful crypto workflow aggregator that pulls in news, on-chain data, social media, web3 services and much more into one customizable UI.",
+  "website": "https://app.alphaday.com/b/dfinity/",
+  "logo": "/img/showcase/alphaday_logo.webp",
```
Comment (jennifertrin) [inline showcase.json:3304]: Can you please add a logo file to this path?

**Review by jennifertrin — CHANGES_REQUESTED** [review]
We reran the CLA. Can you please review the comments related to adding files to the correct path?

File: showcase.json (hunk 1 above)
Comment (Mikael337) [reply showcase.json]: Done

File: showcase.json (hunk 2 above)
Comment (Mikael337) [reply showcase.json]: Done

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**